.circleci/config.yml

```diff
@@ -1,9 +1,10 @@
 version: 2 # use CircleCI 2.0
 jobs: # a collection of steps
   build: # runs not using Workflows must have a `build` job as entry point
-    parallelism: 1 # run three instances of this job in parallel
+    context: appfolio_test_context
+    parallelism: 1
     docker: # run the steps with Docker
-      - image: circleci/ruby:2.4.5-jessie-node # ...with this image as the primary container; this is where all `steps` will run
+      - image: cimg/ruby:2.5.8-node # ...with this image as the primary container; this is where all `steps` will run
```

Comment:
Does the comment still apply?

Reply:
I believe so.
Although these comments are not that useful.
I think they made sense when CircleCI 2.0 was released, which moved from VMs to containers.